catalog.bom

```diff
@@ -0,0 +1,17 @@
+brooklyn.catalog:
+  version: "2.3.0-SNAPSHOT" # BROOKLYN_ETCD_VERSION
+  iconUrl: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-glyph-color.png
+  description: |
+    CoreOS etcd is an open-source distributed key-value store that serves as
+    the backbone of distributed systems by providing a canonical hub for
+    cluster coordination and state management.
+  publish:
+    license_code: APACHE-2.0
+    license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+    overview: README.md
+
+  brooklyn.libraries:
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.brooklyn.etcd&a=brooklyn-etcd&v=2.3.0-SNAPSHOT" # BROOKLYN_ETCD_VERSION
```

Comment:
This URL will be wrong when we do a release build (e.g. switch from 2.3.0-SNAPSHOT to 2.3.0, then it will be looking in `r=snapshots` for it. Not sure what we should work around that! For now, it could just be a documented part of the release process for etcd - but if so, it needs to be documented.